REVIEW_GUIDELINES.md

```diff
@@ -76,6 +76,29 @@ image:
 * The use of the `default` function should be avoided if possible in favor of defaults in `values.yaml`.
 * It is usually best to not specify defaults for resources and to just provide sensible values that are commented out as a recommendation, especially when resources are rather high. This makes it easier to test charts on small clusters or Minikube. Setting resources should generally be a conscious choice of the user.
 
+## Persistence
+
+* Persistence should be turned off by default
+* PVCs should support specifying an existing claim
+* Storage class should be empty by default
```

Comment:
Can we elaborate here and mention it should "be empty by default, to use the default storage class"?